kong/core/globalpatches.lua

```diff
@@ -124,9 +124,13 @@ return function(options)
         end
         return true
       end
-      function SharedDict:incr(key, value)
+      function SharedDict:incr(key, value, init)
         if not self.data[key] then
-          return nil, "not found"
+          if not init then
+            return nil, "not found"
+          else
+            self.data[key].value = init
+          end
```

Comment:
yep, you're right. Sorry for the noise.

I find that so counterintuitive that my brain only read what is was already thinking 😕